### PR TITLE
mcrcon: init at 0.0.5

### PR DESCRIPTION
**Pull request by dermetfan**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/tools/networking/mcrcon/default.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    owner = "Tiiffi";
+    repo = "mcrcon";
+    rev = "v${version}";
+    sha256 = "1pwr1cjldjy8bxqpp7w03nvdpw8l4vqfnk6w6b3mf0qpap1k700z";
+  };
+
+  buildInputs = [ gcc ];
+
+  buildPhase = ''
+    gcc mcrcon.c -o mcrcon
```
Comment (Mic92) [inline pkgs/tools/networking/mcrcon/default.nix]:
```
$CC 
```

might be more portable.

File: pkgs/tools/networking/mcrcon/default.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 755 mcrcon $out/bin
```
Comment (Mic92) [inline pkgs/tools/networking/mcrcon/default.nix]: installPhase = ''
  install -D -m755 mcrcon $out/bin/mcron
'';

File: pkgs/tools/networking/mcrcon/default.nix [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  version = "0.0.5";
+
+  src = fetchFromGitHub {
+    owner = "Tiiffi";
+    repo = "mcrcon";
+    rev = "v${version}";
+    sha256 = "1pwr1cjldjy8bxqpp7w03nvdpw8l4vqfnk6w6b3mf0qpap1k700z";
+  };
+
+  buildInputs = [ gcc ];
```
Comment (Mic92) [inline pkgs/tools/networking/mcrcon/default.nix]: are you sure there is a hard dependency on gcc? If not you can just remove this line and use the command as shown above.

File: pkgs/tools/networking/mcrcon/default.nix [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  buildPhase = ''
+    gcc mcrcon.c -o mcrcon
+  '';
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m 755 mcrcon $out/bin
+  '';
+
+  phases = "unpackPhase buildPhase installPhase fixupPhase";
```
Comment (Mic92) [inline pkgs/tools/networking/mcrcon/default.nix]: These are all default phases. You can remove this.

Comment (dermetfan) [inline pkgs/tools/networking/mcrcon/default.nix]: Thanks for the review! I hoped excluding some could speed up the build. Not a good practice?

Comment (joachifm) [inline pkgs/tools/networking/mcrcon/default.nix]: @dermetfan disabling problematic phases as necessary is more robust, generally.

**Pull request merged**